<ui version="4.0" >
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7" >
   <item>
    <widget class="QTabWidget" name="tabs_" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="DisplayTab_" >
      <attribute name="title" >
       <string>Display</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QCheckBox" name="hlCurLineCheck_" >
         <property name="text" >
          <string>Highlight current line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="visibleWSpaceCheck_" >
         <property name="text" >
          <string>Show white space markers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="marginLineNumbersCheck_" >
         <property name="text" >
          <string>Show line numbers in margin</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" >
         <item>
          <widget class="QCheckBox" name="eolMarkerCheck_" >
           <property name="text" >
            <string>Show end-of-line marker</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Column:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="eolMarkerSpin_" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="indentTab_" >
      <attribute name="title" >
       <string>Indentation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QCheckBox" name="indentTabsCheck_" >
         <property name="text" >
          <string>Indent with tabs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" native="1" name="widget" >
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Tab Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tabWidthSpin_" />
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>173</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Language Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Language</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="indentLanguageCombo_" />
            </item>
            <item>
             <spacer name="horizontalSpacer_5" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox" >
            <property name="text" >
             <string>Maintain indentation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2" >
            <property name="text" >
             <string>Indent by opening braces</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_3" >
            <property name="text" >
             <string>Indent by closing braces</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>132</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="styleTab_" >
      <attribute name="title" >
       <string>Styles</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Styles</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeView" name="styleView_" >
             <property name="headerHidden" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5" >
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Properties</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeView" name="propView_" >
             <property name="alternatingRowColors" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" >
             <item>
              <spacer name="horizontalSpacer_3" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="applyInheritButton_" >
               <property name="toolTip" >
                <string>Make child styles inherit this property</string>
               </property>
               <property name="text" >
                <string>&amp;Apply to Sub-Styles</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QPushButton" name="importButton_" >
           <property name="toolTip" >
            <string>Load styles from an external file</string>
           </property>
           <property name="text" >
            <string>&amp;Import...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportButton_" >
           <property name="toolTip" >
            <string>Store styles in an external file</string>
           </property>
           <property name="text" >
            <string>&amp;Export...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>168</width>
             <height>28</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton_" >
           <property name="toolTip" >
            <string>Revert to default styles</string>
           </property>
           <property name="text" >
            <string>&amp;Reset Styles</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="miscTab_" >
      <attribute name="title" >
       <string>Miscellaneous</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" >
         <item>
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Default Vi Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="viModeCombo_" />
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>337</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>tabs_</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>251</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel" >
     <x>359</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>185</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel" >
     <x>355</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indentTabsCheck_</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>49</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel" >
     <x>166</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indentLanguageCombo_</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>indentLanguageChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>119</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel" >
     <x>429</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton_</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>resetStyles()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>580</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel" >
     <x>443</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyInheritButton_</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>applyInheritance()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>538</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel" >
     <x>598</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importButton_</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>importStyles()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>74</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel" >
     <x>4</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportButton_</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>exportStyles()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>159</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel" >
     <x>101</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eolMarkerCheck_</sender>
   <signal>toggled(bool)</signal>
   <receiver>eolMarkerSpin_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>108</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel" >
     <x>542</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>indentLanguageChanged(int)</slot>
  <slot>resetStyles()</slot>
  <slot>applyInheritance()</slot>
  <slot>importStyles()</slot>
  <slot>exportStyles()</slot>
 </slots>
</ui>
